### PR TITLE
pointing on new geojson 4 tooltip

### PR DESCRIPTION
**Pull request by rebert**

### CODE REVIEW

**Comment by ltclm** [comment]
i have just noticed that the weibull diagram in the extended tooltip is empty, with the new json export:

https://mf-chsdi3.dev.bgdi.ch/ltclm/rest/services/ech/MapServer/ch.bfe.windenergie-geschwindigkeit_h50/2528_215/extendedHtmlPopup?lang=de

File: chsdi/views/features.py [hunk 1]
```diff
@@ -269,6 +267,10 @@ def _identify_grid(params, layerBodIds):
         grid = Grid(gridSpec.get('extent'),
                     gridSpec.get('resolutionX'),
                     gridSpec.get('resolutionY'))
+        if params.srid == 2056 and gridSpec.get('srid') == '21781':
```
Comment (rebert) [inline chsdi/views/features.py]: @procrastinatio works for GeoJson Grids in LV95 now. But not yet in webmercator... and even now my elif dows look ugly. Any idea?

**Comment by danduk82** [comment]
@rebert, I test it this afternoon

**Comment by ltclm** [comment]
@rebert as discussed on the phone, please test the new code with one of the old grid layers.
thank you!

**Comment by rebert** [comment]
@ltclm Thanks for your testing-idea! there are two other layers: https://mf-chsdi3.dev.bgdi.ch/ltrea_winde_geschw/shorten/7fca8e6ae2 and they seem to work

File: chsdi/views/features.py (hunk 1 above)
Comment (danduk82) [reply chsdi/views/features.py]: ping @procrastinatio

**Comment by AFoletti** [comment]
OK to merge

**Pull request merged**